adapters/discord/src/message.ts

```diff
@@ -300,7 +301,15 @@ export class DiscordMessageEncoder<C extends Context = Context> extends MessageE
     } else if (type === 'audio') {
       await this.sendAsset('file', attrs, {
         ...this.addition,
-        content: this.buffer.trim(),
+        content: '',
+        attachments: [
+          {
+            waveform: '', // base64 encoded bytearray representing a sampled waveform
```

Comment:
Should the `waveform` be required here to be calculated from the wave ?

From Discord's doc:
> The `waveform` is intended to be a preview of the entire voice message, with 1 byte per datapoint encoded in base64. Clients sample the recording at most once per 100 milliseconds, but will downsample so that no more than 256 datapoints are in the waveform.

Reply:
It is also possible to send without calculation.